### ISSUE TITLE
Make dropdown menus float above surrounding content

### ISSUE THREAD

**Issue by Zitzak** [PrimeDAO/prime-launch-dapp#177]
Make dropdown menus float above surrounding content

**Comment by renc-curvelabs** [comment]
Optional

**Comment by BartuCulha** [comment]
https://launch.prime.xyz/newSeed
Dropdown works fine on Testnet but not on Mainnet. Are we going to update Mainnet after Doug's return?

**Comment by giladt** [comment]
> https://launch.prime.xyz/newSeed
> Dropdown works fine on Testnet but not on Mainnet. Are we going to update Mainnet after Doug's return? @giladt

All latest changes are not yet deployed to Mainnet. It will have to wait for Doug's return...

**Comment by dkent600** [comment]
Yeah, I’m not deploying regularly to mainnet.  When I do I am always clear in the TG announcements.  If you need a mainnet deploy for some reason, ask and I can do.  If urgent I can do before I get back from vaca
